app/src/test/kotlin/com/wire/android/ui/home/conversations/MessageComposerViewModelTest.kt

```diff
@@ -525,4 +525,40 @@ class MessageComposerViewModelTest {
             assertInstanceOf(SelfDeletionTimer.Enabled::class.java, viewModel.messageComposerViewState.value.selfDeletionTimer)
             assertEquals(expectedDuration, viewModel.messageComposerViewState.value.selfDeletionTimer.duration)
         }
+
+    @Test
```

Comment:
Can we also have test which will check if mimeType is unknown then it will return orDefault(DEFAULT_FILE_MIME_TYPE) ?

Reply:
The `.getMimeType` is an extension function on `Uri` class and on this test class we are already mocking its logic so I suppose there is no need